src/app/modules/item/components/item-children-list/item-children-list.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { ItemTypeCategory } from '../../../../shared/helpers/item-type';
+import { ItemChildWithAdditions } from '../sub-skills/sub-skills.component';
+
+@Component({
+  selector: 'alg-item-children-list',
+  templateUrl: './item-children-list.component.html',
+  styleUrls: [ './item-children-list.component.scss' ],
+})
+export class ItemChildrenListComponent {
```

Comment:
It seems to be specific to skills, so its name should mention skill, otherwise we'll try to open this file each time we want to change the "regular" item children list which is "chapter-children" I think.
Maybe just `SkillChildrenListComponent`?

Reply:
Fixed